src/app/pages/institution-list/institution-list.component.html

```diff
@@ -0,0 +1,6 @@
+<app-institution></app-institution>
```

Comment:
Teniendo la lista de instituciones, hacemos quedaría así:
`<app-institution *ngfor="let institution of institutions" [institution]="institution"></app-institution>`

Entonces estarías iterando la lista y usando el componente una sola vez.

En el componente institution deberías usar ahora el modelo de Institution también, y en vez de pasar por parámetro en el HTML `{{name}}`, vas a pasar `{{institution.name}}`

Cualquier duda avísame y si queres lo vemos juntos, pero vas a ver que con esto adelantamos un montón 👍